src/yang/ietf-tcp.yang

```diff
@@ -319,6 +245,78 @@ module ietf-tcp {
     reference "RFC 8312: CUBIC for Fast Long-Distance Networks";
   }
 
+  // Important global TCP stack parameters
+
+  grouping global {
+    leaf tcp-mss {
+      type uint16;
+      description
+        "Sets the max segment size for TCP connections.";
+    }
+
+    leaf tcp-mtu-discovery {
+      type boolean;
+      default false;
```

Comment:
I noticed that this is the only leaf which is defined as 'config false'. I am no expert, so I assuming that all the other leafs in this grouping is something that can be set by the NETCONF/RESTCONF application.

Another naive question. Also are all these attributes set per connection or can anyone be set at a global level?

Reply:
Regarding 1/

This is *default* false, not config false. Many stacks disable PMTU Discovery by default and their behavior is thus more consistent than for other stacks. But as it is difficult to provide defaults for some other attributes, I have now removed the default

Regarding 2/

Different TCP stacks differ ;-) However, the more common solution is probably to configure this globally for all connections, and this is why I'd suggest to model it that way. If a stack allows setting this or a sub-set per connection, the model could be augmented. For most of the attibutes there is not much benefit in setting it per connection.